Sources/UtiliKit/Obfuscation/ObfuscatedKey.swift

```diff
@@ -0,0 +1,109 @@
+//
+//  ObfuscatedKey.swift
+//  UtiliKit-iOS
+//
+//  Created by Russell Mirabelli on 7/23/19.
+//  Copyright © 2019 CocoaPods. All rights reserved.
+//
+
+// Inspired by https://github.com/onmyway133/Arcane/blob/master/Sources/Arcane/Obfuscator.swift
+
+import Foundation
+
+/// One should never commit keys directly into one's source; this is an unsafe practice.
+/// When it is impossible to avoid doing so, the key should at least be obfuscated.
+/// By using the ObfuscatedKey struct, you can build a human-readable key that nonetheless
+/// will not appear simply by running "strings" against your compiled code, and will even
+/// not appear as a string within your source code.
+struct ObfuscatedKey {
+    private let _value: String
+
+    init() {
+        self._value = ""
+    }
+
+    init(_ value: String) {
+        self._value = value
+    }
+
+    var value: String {
+        get {
+            return _value
+        }
+    }
+
+    // codebeat:disable[TOO_MANY_FUNCTIONS]
+    var A: ObfuscatedKey { return ObfuscatedKey(_value + "A") }
```

Comment:
It might make sense to throw these into a `public extension` to alleviate the need to define each of these as `public var ...`

Reply:
Nice way to avoid all the publics. Done, and thanks!